### PR TITLE
Job definition filter implementation

### PR DESCRIPTION
**Pull request by dpb587**
This is an implementation of what @mrdavidlaing proposed in his [comment](https://github.com/logsearch/logsearch-boshrelease/issues/9#issuecomment-41587591).

Users are able to update the `logstash_parser.filters` config with URLs of their tar/tar.gz/zip files. Files are only downloaded once and are extracted in order of appearance within the array value. This means a later file could overwrite an earlier, upstream version if using the exact same name (which I consider a feature). Once extracted, they're loaded into the logstash configuration file still in order of priority based on the name.

### CODE REVIEW

File: jobs/log_parser/spec [hunk 1]
```diff
@@ -2,7 +2,6 @@
 name: log_parser
 packages: 
   - logstash
-  - logstash-filters-common
```
Comment (mrdavidlaing) [inline jobs/log_parser/spec:5]: @dpb587 I think that the common filters should be part of the base boshrelease; so you don't have an external dependancy "out the box"

**Comment by dpb587** [comment]
So we're going to maintain both boshrelease filter configuration _and_ job filter configuration?

**Comment by mrdavidlaing** [comment]
No.  Let me show you what I mean

**Comment by mrdavidlaing** [comment]
Relocated to target `develop`

**Pull request closed without merging**